src/arm/cpu.rs

```diff
@@ -0,0 +1,30 @@
+use cfg_if::cfg_if;
+use libc;
+
+pub const DAV1D_ARM_CPU_FLAG_NEON: libc::c_uint = 1 << 0;
+pub const NEON_HWCAP: libc::c_ulong = 1 << 12;
+
+#[no_mangle]
+#[cold]
+pub unsafe extern "C" fn dav1d_get_cpu_flags_arm() -> libc::c_uint {
+    let mut flags = 0;
+
+    cfg_if! {
+        if #[cfg(any(
+            target_arch = "aarch64",
+            target_os = "windows",
+            target_os = "macos"
+        ))] {
+            flags |= DAV1D_ARM_CPU_FLAG_NEON;
+        } else if #[cfg(target_os = "android")] {
+            // TODO: Support Android by parsing `/proc/cpuinfo` the way the original C does.
+            todo!("Android is not yet supported")
+        } else {
```

Comment:
nit: In the C version, 
- this path is guarded on `ARCH_ARM` which I think means 32-bit arm (so `target_arch  = "arm"`), and
- it comes before the android path

Would it be better to guard it on 32-bit arm like the original code and put the test before android? I'm not 100% convinced this even matters, so you decide what makes sense here.

Reply:
Nah, you're right. I misread the original C so what you're suggesting is more correct. Fixed.